include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -103,6 +82,53 @@ struct CShuffleEpilogue
         return MaxVectorStoreSize / sizeof(ODataType);
     }
 
+    /**
+     * @brief Shuffle tile configuration parameters
+     *
+     * @details These parameters control the number of XDL tiles processed per wave in each shuffle
+     * iteration:
+     * - NumMXdlPerWavePerShuffle: Number of XDL tiles in M dimension processed per wave
+     * - NumNXdlPerWavePerShuffle: Number of XDL tiles in N dimension processed per wave
+     */
+    static constexpr auto ShuffleTileTuple = [] {
+        constexpr index_t vecPerThread = kMPerXdl * kNPerXdl / get_warp_size();
```

Comment:
Please use snake case naming for non static variables - please keep style coherent throughout the file.